lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -48,13 +48,21 @@ export const BrowserAuthErrorMessage = {
         code: "user_cancelled",
         desc: "User cancelled the flow."
     },
-    popupWindowTimeoutError: {
-        code: "popup_window_timeout",
-        desc: "Popup window token acquisition operation failed due to timeout."
+    monitorWindowTimeoutError: {
+        code: "monitor_window_timeout",
+        desc: "Token acquisition operation failed - monitorWindowForHash timed out."
```

Comment:
`monitorWindowForHash` is an implementation detail and will be meaningless to consumers, in my opinion.

Reply:
Added it here to differentiate between monitorWindowForHash vs monitorFrameForHash for MSAL devs. If you have a better error message please suggest.

Reply:
"Token acquisition in popup failed due to timeout."

Reply:
done.